dotnet-authserver/src/TeacherIdentity.AuthServer/Pages/Admin/User.cshtml

```diff
@@ -56,6 +57,25 @@
                     </govuk-summary-list-row-actions>
                 }
             </govuk-summary-list-row>
+            <govuk-summary-list-row>
+                <govuk-summary-list-row-key>Merged user IDs</govuk-summary-list-row-key>
+                <govuk-summary-list-row-value>
+                    @if (Model.MergedUserIds.IsNullOrEmpty())
+                    {
+                        <p>None</p>
+                    }
+                    else
+                    {
+                        foreach (var userId in Model.MergedUserIds!)
+                        {
+                            <p>@userId</p>
```

Comment:
This displays the full list, perhaps we want to be able to collapse/expand the list?

Reply:
Probably ok for now. Hopefully this list will be short!